src/fairchem/core/common/utils.py

```diff
@@ -999,34 +1024,70 @@ class _TrainingContext:
             task_name = "s2ef"
         elif trainer_name in ["energy", "equiformerv2_energy"]:
             task_name = "is2re"
+        elif "multitask" in trainer_name.lower():
```

Comment:
can we get rid of this .lower(), this makes it ambiguous on how to specify the trainer name